### PR TITLE
[GH 59] Enable/disable console logging

### PR DESCRIPTION
**Pull request by Tianqi-Tang**
Add an API on storops module to enabled and disable the console logging

### CODE REVIEW

File: test/test_storops.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+# Copyright (c) 2015 EMC Corporation.
```
Comment (peter-wangxu) [inline test/test_storops.py]: 2016

File: storops/__init__.py [hunk 2]
```diff
@@ -15,9 +15,29 @@
...
 __author__ = 'Cedric Zhuang'
+
+
+def enable_log(level=logging.DEBUG):
+    """Enable console logging.
+
+    This is a utils method for try run with storops.
+    """
+    log = logging.getLogger(__name__)
+    log.setLevel(level)
```
Comment (jealous) [inline storops/__init__.py]: Is this effective for all loggers under storops?

Comment (Tianqi-Tang) [inline storops/__init__.py]: Yes, it is for the whole hierarchy

File: test/test_storops.py (hunk 1 above)
Comment (Tianqi-Tang) [inline test/test_storops.py]: Done

**Pull request merged**